packages/data-context/src/actions/ProjectActions.ts

```diff
@@ -310,6 +310,10 @@ export class ProjectActions {
     this.ctx.emitter.specsChange()
   }
 
+  refreshSpecs (specs: FoundSpec[]) {
+    this.ctx.lifecycleManager.git?.setSpecs(specs.map((s) => s.absolute))
+  }
```

Comment:
Nice, can we get an e2e test that covers it? Would prob go in `specs_list_actual_git_repo.cy.ts`

Reply:
I have one coming after the UI updates for https://cypress-io.atlassian.net/browse/CLOUD-604
![image](https://user-images.githubusercontent.com/15657366/167020455-a871f89e-a10b-416f-b797-eb94f206de03.png)

Reply:
We could add one with this PR and then just adjust it in the next PR so we don't miss it. We typically like to have a test covering behavior changes in the PR that introduces them